pytorch_lightning/lite/lite.py

```diff
@@ -86,6 +86,7 @@ def __init__(
             devices=devices,
             tpu_cores=tpu_cores,
             ipus=None,
+            hpus=None,
```

Comment:
I don't think the logic for supporting Lite with Habana has been added?

Reply:
Lite support is not yet added. Is it a mandatory one for new accelerator support ?

Reply:
Based on slack comments. We'll add support for this in a subsequent PR